test/CompressTest.php

```diff
@@ -22,6 +23,7 @@
 use function uniqid;
 use function unlink;
 
+/** @psalm-import-type AdapterType from Compress */
```

Comment:
```suggestion
/** @psalm-import-type AdapterType from CompressFilter */
```
And the extra import in line 8 is not needed.